client/docs/config.json

```diff
@@ -1,21 +1,25 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "AKASH - GRPC Gateway",
+    "title": "AKASH - gRPC Gateway docs",
+    "description": "A REST interface for state queries, legacy transactions",
     "version": "1.0.0"
   },
   "apis": [
     {
-      "url": "./akash/deployment/v1beta1/query.swagger.json"
+      "url": "./tmp-swagger-gen/akash/deployment/v1beta1/query.swagger.json"
     },
     {
-      "url": "./akash/market/v1beta1/query.swagger.json"
+      "url": "./tmp-swagger-gen/akash/market/v1beta1/query.swagger.json"
     },
     {
-      "url": "./akash/provider/v1beta1/query.swagger.json"
+      "url": "./tmp-swagger-gen/akash/provider/v1beta1/query.swagger.json"
```

Comment:
what is the `tmp-` prefix about?  can we not check all of these artifacts in?  if we're packing it with statik I'd prefer to only check in the packed version.

Reply:
This is just a temporary folder in which we generate swagger docs of each module. We will combine all those swagger json files into one doc using this config.json. Once after combining all, we will delete this temp folder. 

[This](https://github.com/ovrclk/akash/pull/831/files#diff-11f7032d70c63ebaa71cf65c25ec2a66f4fba95d9a4472caaee7829dba417a82) script creates temp folder, combines all docs and delete temp folder.

Reply:
I see, thank you